py_getter/__init__.py

```diff
@@ -120,7 +116,8 @@ def cli():
     parser.add_argument("URI")
     parser.add_argument("PATH")
     parser.add_argument("--refresh", action="store_true")
+    parser.add_argument("--s3_endpoint_url")
```

Comment:
Just a nit to use a hyphen here as separator for cli args...

```suggestion
    parser.add_argument("--s3-endpoint-url")
```